allofplos/transformations.py

```diff
@@ -118,8 +170,7 @@ def doi_to_url(doi, plos_network=False):
     :param doi: full unique identifier for a PLOS article
     :return: online location of a PLOS article's XML
     """
-    URL_TMP = INT_URL_TMP if plos_network else EXT_URL_TMP
-    return URL_TMP.format(doi)
+    return ''.join([get_base_page(doi_to_journal(doi)), 'article/file?id=', doi, url_suffix])
```

Comment:
Better to break this up; it’s pretty complicated.

Reply:
fixed in b59773b